### PR TITLE
Fix __isabstractmethod__ for class/static methods and properties

### PR DESCRIPTION
**Pull request by chrismoradi**
Includes a minor departure from CPython implementation of `inspect.isabstract` because TPFLAGS_IS_ABSTRACT is not set in `obj.__flags__`. Not sure if this is a problem but happy to revert this change if it is.

### CODE REVIEW

File: Lib/inspect.py [hunk 1]
```diff
@@ -300,16 +300,16 @@ def isroutine(object):
 
 def isabstract(object):
     """Return true if the object is an abstract base class (ABC)."""
+    # TODO: RUSTPYTHON
```
Comment (youknowone) [inline Lib/inspect.py:303]: How about leaving the original code as commented-out and put this comments on line 305? So future contributor can notice what's changed in the source code.

File: vm/src/builtins/property.rs [hunk 2]
```diff
@@ -217,6 +217,32 @@ impl PyProperty {
         }
         .into_ref_with_type(vm, TypeProtocol::clone_class(&zelf))
     }
+
+    #[pyproperty(magic)]
+    fn isabstractmethod(&self, vm: &VirtualMachine) -> PyObjectRef {
+        let getter_abstract = match self.getter.read().to_owned() {
```
Comment (youknowone) [inline vm/src/builtins/property.rs:223]: @coolreader18 is `to_owned` the best way to handle this?

File: Lib/inspect.py (hunk 1 above)
Comment (chrismoradi) [reply Lib/inspect.py]: Sure! Fixed this.

**Review by youknowone — APPROVED** [review]

File: vm/src/builtins/property.rs (hunk 2 above)
Comment (youknowone) [reply vm/src/builtins/property.rs:223]: @coolreader18 do you have any idea about this? or i will merge this PR

Comment (chrismoradi) [reply vm/src/builtins/property.rs:223]: @youknowone & @coolreader18, Just wanted to check in on this PR. Should I work on trying to remove the `to_owned()`?

**Comment by youknowone** [comment]
I am sorry to keep this unmerged for long time!

**Pull request merged**